add mongoDB tag (#1583)
fireship-io/fireship.io: content/tags/mongoDB/_index.md

---
title: MongoDB

> description: MongoDB is a source-available cross-platform document-oriented NoSQL database program where it uses JSON-like documents with optional schemas.

Reviewer: ```suggestion
description: MongoDB is a source-available cross-platform document-oriented NoSQL database program that uses JSON-like documents with optional schemas.
```
